cpp/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(root_directory ${PROJECT_BINARY_DIR})
 #
 
 if(DEBUG)
-  add_definitions(-DDEBUG)
+  add_definitions(-DGLUTEN_PRINT_DEBUG)
```

Comment:
Do you think we need to set two debug option?

Reply:
No. DEBUG is cmake flag and GLUTEN_PRINT_DEBUG is the compilation flag. I just found the DEBUG compilation flag also used in velox. Change it to another one can avoid conflict.